govc/vm/snapshot/tree.go

```diff
@@ -161,8 +174,15 @@ func (cmd *tree) Run(ctx context.Context, f *flag.FlagSet) error {
 	}
 
 	cmd.info = o.Snapshot
+	cmd.layout = o.LayoutEx
+
+	// @dougm: This was in the original code snippets shared, but I assume
+	// isn't desired in a final PR?
+	//
+	// size := SnapshotSize(o.Reference(), nil, o.LayoutEx, true)
```

Comment:
This comment block is what I made reference to in my last comment.

Reply:
Right, this can be removed.